### ISSUE TITLE
Using iframes

### ISSUE THREAD

**Issue by ronidavelman** [STRML/react-grid-layout#788]
- [x] Question

Using iframes inside of the draggable elements creates quite a noticeable lag when resizing. Are there any extra steps necessary to make this smoother?

**Comment by Circle6** [comment]
For anyone else who may have had issues with this like I did I found the problem and the solution.

The issue is not that your draggable element is "lagging", but that when resizing and moving your mouse over the iframe your mouse events are now being hijacked by the iframe! This can mean that your resizeable element doesn't receive events to resize or mouseup when you're done dragging.

The easy solution is to add an onResizeStart and onResizeEnd function to your grid that sets a "resizing" state when an element is being resized. Pass this state down to the element itself and when your iframe elements are being resized, render an invisible div on top of your iframe to prevent the events from being hijacked.

Voila! No more lag! Hope this helps :)
